scripts/hudson/narayana.sh

```diff
@@ -185,6 +185,13 @@ function init_test_options {
         else
           export COMMENT_ON_PULL=""
         fi
+    elif [[ $PROFILE == "DB_TESTS" ]]; then
+        if [[ ! $PULL_DESCRIPTION_BODY == *!LRA* ]]; then
```

Comment:
Shouldn't this be `... ! $PULL_DESCRIPTION_BODY == *!DB_TESTS*`

Reply:
Good catch - thanks